src/components/LearningCard.spec.js

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react'
+import LearningCard from './LearningCard'
+
+describe('Card', () => {
+  it('renders', () => {
+    render(
+      <LearningCard
+        name="Name of the emotion"
+        description="Description of the Emotion"
+        tags="Tag of the emotion"
```

Comment:
Tags is an array. You need to add an array here. And you are also missing the `color` property.
```suggestion
        tags={["ängstlich", "Schreck", "zittern", "Furcht"]}
```